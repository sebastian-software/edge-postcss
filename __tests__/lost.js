import postcss from "postcss"
import loadConfig from "postcss-load-config"

test("Render Lost Grid", () => {
  let css = "h1 { lost-column: 3/12 }"

  return loadConfig({ map: "inline" }).then(({ plugins, options }) => {
    return postcss(plugins)
      .process(css, options)
      .then((result) => expect(result.css).toMatchSnapshot())
  })
})
