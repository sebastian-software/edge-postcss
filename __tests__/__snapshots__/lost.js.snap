// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Lost Grid 1`] = `
"h1{width:calc(24.975% - 22.5px)}h1:nth-child(n){float:left;margin-right:30px;clear:none}h1:last-child{margin-right:0}h1:nth-child(12n){margin-right:0;float:right}h1:nth-child(12n+1){clear:both}
/*# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIjxpbnB1dCBjc3MgMT4iXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBQUEsR0FBSyw0QkFBaUIsQ0FBRSxBQUF4QixnQkFBSyxXQUFpQixBQUFqQixrQkFBaUIsQUFBakIsVUFBaUIsQ0FBRSxBQUF4QixjQUFLLGNBQWlCLENBQUUsQUFBeEIsa0JBQUssZUFBaUIsQUFBakIsV0FBaUIsQ0FBRSxBQUF4QixvQkFBSyxVQUFpQixDQUFFIiwiZmlsZSI6InRvLmNzcyIsInNvdXJjZXNDb250ZW50IjpbImgxIHsgbG9zdC1jb2x1bW46IDMvMTIgfSJdfQ== */"
`;
